presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2194,7 +2194,7 @@ public void createView(ConnectorSession session, ConnectorTableMetadata viewMeta
         Table table = tableBuilder.build();
         PrincipalPrivileges principalPrivileges = buildInitialPrivilegeSet(session.getUser());
 
-        MetastoreContext metastoreContext = new MetastoreContext(session.getIdentity(), session.getQueryId(), session.getClientInfo(), session.getSource());
+        MetastoreContext metastoreContext = new MetastoreContext(session.getIdentity(), session.getQueryId(), session.getClientInfo(), session.getSource(), HiveUtil.getMetastoreHeader(session));
```

Comment:
please use getMetastoreHeaders() in HiveSessionProperties here and all other places

Reply:
✅